sycl/include/CL/sycl/detail/pi.h

```diff
@@ -649,6 +649,7 @@ static const uint8_t PI_DEVICE_BINARY_OFFLOAD_KIND_SYCL = 4;
 /// PropertySetRegistry::SYCL_SPECIALIZATION_CONSTANTS defined in
 /// PropertySetIO.h
 #define PI_PROPERTY_SET_SPEC_CONST_MAP "SYCL/specialization constants"
+#define PI_PROPERTY_SET_DEVICELIB_REQ_MASK "SYCL/devicelib req mask"
```

Comment:
since you introduced LLVM/RT shared header, these two declarations can be moved there

Reply:
Done.